shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -1358,6 +1361,9 @@ public void revertImageView(@NonNull Runnable onDone) {
       Log.v(TAG, "Tried to revert the image view, but no previous surface was used.");
       return;
     }
+    if (previousRenderSurface == flutterImageView) {
```

Comment:
To prevent executing the following code when the previous render surface and the current are the same. I'm not sure how it happens so I just add this to make it safe to call this method.

Reply:
I think we need to understand what is happening better here.

Reply:
On second thought, this will not happen. I undo this.